dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -491,6 +580,22 @@ void resolveFeaturesAsSet() {
         }
     }
 
+    /**
+     * Record Versionless Messages and Issues
+     *
+     * @param result
+     * @param resolverRepo
+     * @param requestedFeatures
+     */
+    private void recordVersionless(Result result, Repository resolverRepo) {
+
+        resolvedPlatforms = result.getResolvedPlatforms();
+        missingPlatforms = result.getMissingPlatforms();
+        if (resolvedPlatforms.isEmpty() || !missingPlatforms.isEmpty())
+            hasVersionlessIssue = true;
+
+    }
```

Comment:
Contrary to the javadoc, this method doesn't appear to record anything or do anything to do with messages.

Reply:
removed the hasVersionlessIssie - it simply sets the result attributes used for for passing to the exception.